src/Microsoft.Health.Dicom.Api/Extensions/HttpResponseExtensions.cs

```diff
@@ -28,7 +28,7 @@ public static void AddLocationHeader(this HttpResponse response, Uri locationUrl
         EnsureArg.IsNotNull(response, nameof(response));
         EnsureArg.IsNotNull(locationUrl, nameof(locationUrl));
 
-        response.Headers.Add(HeaderNames.Location, locationUrl.IsAbsoluteUri ? locationUrl.AbsoluteUri : GetRelativeUri(locationUrl));
+        response.Headers.Append(HeaderNames.Location, locationUrl.IsAbsoluteUri ? locationUrl.AbsoluteUri : GetRelativeUri(locationUrl));
```

Comment:
This is a new analyzer that recommends appending headers as opposed invoking Add from the dictionary interface. That is because Append will accommodate multiple values for the same key whereas Add will throw an error